packages/stream_chat_flutter/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - [[#1740]](https://github.com/GetStream/stream-chat-flutter/issues/1740) Fixed
   Plugin `video_player:macos` has multiple possible implementations.
+- [[#1759]](https://github.com/GetStream/stream-chat-flutter/issues/1759) Fixed
```

Comment:
I think 6.11.0 is already released. Can we add this to an upcoming release in the changelog?